### ISSUE TITLE
fatal: Remote branch sgx_2.11_reproducible not found in upstream origin

### ISSUE THREAD

**Issue by sbellem** [intel/linux-sgx#615]
The branch ` sgx_2.11_reproducible` is missing ...

https://github.com/intel/linux-sgx/blob/33f4499173497bdfdf72c5f61374c0fadc5c5365/linux/reproducibility/build_and_launch_docker.sh#L182

For instance, (on `master` branch), when running the `build_and_launch_docker.sh` bash script:

```shell
$ ./build_and_launch_docker.sh -t sdk
...
Cloning into '/home/teeman/intel/linux-sgx/linux/reproducibility/code_dir/sgx'...
fatal: Remote branch sgx_2.11_reproducible not found in upstream origin
```

**Comment by lzha101** [comment]
This should be by a publishing mistake.  Just created a PR https://github.com/intel/linux-sgx/pull/618 to recover the build scripts on master branch, although it is recommended to use the official reproducible branch https://github.com/intel/linux-sgx/tree/sgx_2.10_reproducible for reproducibility verification.

**Comment by sbellem** [comment]
Ok, thanks!

**Comment by sbellem** [comment]
Fixed in #618. Closing.
